test/e2e/framework.go

```diff
@@ -100,8 +100,8 @@ func (f *Framework) WaitForPodRunning(podName string) error {
 }
 
 // Runs the given pod and verifies that its output matches the desired output.
-func (f *Framework) TestContainerOutput(scenarioName string, pod *api.Pod, expectedOutput []string) {
-	testContainerOutputInNamespace(scenarioName, f.Client, pod, expectedOutput, f.Namespace.Name)
+func (f *Framework) TestContainerOutput(scenarioName string, pod *api.Pod, containerId int, expectedOutput []string) {
+	testContainerOutputInNamespace(scenarioName, f.Client, pod, containerId, expectedOutput, f.Namespace.Name)
```

Comment:
To save the work to look up trivial container names for different test cases. More intuitive as well, I guess. 
After all, all of the existing test cases involve only one container unless #9234 is merged.